### ISSUE TITLE
Division by float: convert ProdExpr -> Expr

### ISSUE THREAD

**Issue by ramabile** [SCIP-Interfaces/PySCIPOpt#213]
Dear SCIPers,
Following this https://github.com/SCIP-Interfaces/PySCIPOpt/issues/212#issuecomment-435042087 I am here to ask for a clarification.

Consider:
```python
from pyscipopt import Model
model = Model()
x = model.addVar(vtype="c")
print((2*x).__class__.__name__)
# Expr
print((x/2.).__class__.__name__)
# ProdExpr
print((2.*x).__class__.__name__)
# Expr
print((x/2).__class__.__name__)
# ProdExpr
```
It seems like `ProdExpr` is not casted as `Expr` in any case the division is defined. This provokes:
```python
model.setObjective(.5*x) # accepted
model.setObjective(x/2.) # not accepted
# AssertionError: given coefficient is neither Expr or number but ProdExpr
```
Is this a design choice? Is SCIP behaviour?

As a Python user, I would expected a full support of the division of a continuous variable with a float number.
In other cases like integer variable divided by float or continuous variable divided by integer, I would expect a division btw. floats since integers are casted to floats in these cases.
In case of division between an integer variable and an integer number I would keep the `ProdExpr` until there is no support for division with remainders (that is why I wrote [this](https://github.com/SCIP-Interfaces/PySCIPOpt/pull/211/files#diff-3ff183b9d68ae9718e7ecc814d299d67)).

**Comment by fserra** [comment]
this was just a mistake.  the code is assuming that when a user writes `/` then the user means dividing by something that is not just a number. The PR #214 should fix this.

**Comment by ramabile** [comment]
Closed by #214.
